assume-role.sh

```diff
@@ -2,7 +2,7 @@
 set -ue
 
 
-ASSUME_ROLE="$(aws sts assume-role --role-arn arn:aws:iam::991225504892:role/tf-pipeline-role --role-session-name session10)"
+ASSUME_ROLE="$(aws sts assume-role --role-arn arn:aws:iam::991225504892:role/tf-pipeline-role --role-session-name pipeline-session)"
```

Comment:
Could you please check whether the name of the session may contain a hyphen before we merge this one?

Reply:
According to the documentation this should work because the constraints are the following:

```
a string of characters
consisting of upper- and lower-case alphanumeric characters with no
spaces. You can also include underscores or any of the following
characters: =,.@-
```

It is always better to check, if it works IRL :)

Reply:
I checked, it works 👌

Reply:
@tponomarenko thanks!